app/views/shared/_resources_table.html.haml

```diff
@@ -3,8 +3,23 @@
     %tr
       %th #
       - columns.each do |column|
-        %th
-          = resource_class.human_attribute_name(column.to_sym)
+        %th.th-sortable
+          - if (defined? unsortable_columns) && (column.in? unsortable_columns)
+            .text
+              = resource_class.human_attribute_name(column.to_sym)
+          - else
+            - is_current_column = (column.to_s == sort_column)
+            - is_current_asc = (sort_direction == "asc")
+            - new_direction = (is_current_column && is_current_asc) ? "desc" : "asc"
```

Comment:
We want the default to be `desc`, right? In this case, for other columns will be in `asc`.

Reply:
In this case, we want default sort by created_at with newest right? and this shows newest first.